external/netstandard20/netstandard20.depproj

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard;
+      netstandard2.0;
```

Comment:
Yes that is fine as well, although if we do that we should probably map netfx to netfx461.